.npmignore

```diff
@@ -1 +1,6 @@
-.test.js
\ No newline at end of file
+.test.js
+.husky
+.github
+.prettierrc
+jest.config.js
+vendors/
```

Comment:
Updated this to reflect the only necessary files. `vendors/` is no longer needed.